### PR TITLE
words: this title should have been a pun, but I chickened out

### PR DESCRIPTION
**Pull request by catzkorn**
Signed-off-by: Charlotte Brandhorst-Satzkorn <charlotte@tailscale.com>

### CODE REVIEW

**Comment by apenwarr** [comment]
Should we do hen and rooster while we’re here?

File: words/scales.txt [hunk 1]
```diff
@@ -113,6 +113,11 @@ bee
 bearded
 beardie
 pogona
+chicken
```
Comment (danderson) [inline words/scales.txt:116]: These creatures have scales? Wha?

Comment (catzkorn) [reply words/scales.txt:116]: On their legs! I fact checked

Comment (catzkorn) [reply words/scales.txt:116]: > Scaled Skin: This covers the legs and feet and is thought to be vestigial from the dinosaur age. It also provides protection for the legs.

**Review by sailorfrag — APPROVED** [review]
How fowl

**Review by crawshaw — APPROVED** [review]
looking forward to bearded-chicken

**Pull request merged**